lib/extensions/boolean_extension.rb

```diff
@@ -3,11 +3,19 @@ module True
     def to_i
       1
     end
+
+    def to_yesno
+      'Yes'
+    end
   end
 
   module False
     def to_i
       0
     end
+
+    def to_yesno
```

Comment:
[Ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/2220/files#r164720255)